src/libstd/sys/unix/fs.rs

```diff
@@ -149,22 +155,33 @@ impl FileAttr {
 #[cfg(not(any(target_os = "ios", target_os = "macos", target_os = "netbsd")))]
 impl FileAttr {
     pub fn modified(&self) -> io::Result<SystemTime> {
-        Ok(SystemTime::from(libc::timespec {
-            tv_sec: self.stat.st_mtime as libc::time_t,
-            tv_nsec: self.stat.st_mtime_nsec as libc::c_long,
-        }))
+        #[cfg(target_os = "emscripten")]
+        { return Ok(SystemTime::from( self.stat.st_mtim )); }
+        #[cfg(not(target_os = "emscripten"))]
+        {
+            return Ok(SystemTime::from(libc::timespec {
+                tv_sec: self.stat.st_mtime as libc::time_t,
+                tv_nsec: self.stat.st_mtime_nsec as libc::c_long,
+            }));
+        }
     }
 
     pub fn accessed(&self) -> io::Result<SystemTime> {
-        Ok(SystemTime::from(libc::timespec {
-            tv_sec: self.stat.st_atime as libc::time_t,
```

Comment:
Ah I think it's actually getting taken care of in https://github.com/rust-lang/libc/pull/210, once that's merged this can probably just update libc again